flow360/component/flow360_params/flow360_output.py

```diff
@@ -197,8 +184,11 @@ def to_solver(self, params, **kwargs) -> TimeAverageAnimatedOutput:
 class Surface(Flow360BaseModel):
     """:class:`Surface` class"""
 
-    output_fields: Optional[SurfaceOutputFields] = pd.Field(
-        alias="outputFields", displayed="Output fields", default=[]
+    output_fields: Optional[List[str]] = pd.Field(
+        alias="outputFields",
+        displayed="Output fields",
+        default=[],
+        natively_supported=SurfaceFields,
```

Comment:
I think adding custom fields is not good solution. It will not have type hints and we need custom handling of it. Also, pydantic v2 will not support it:
```
pydantic.warnings.PydanticDeprecatedSince20: Using extra keyword arguments on `Field` is deprecated and will be removed. Use `json_schema_extra` instead. (Extra keys: 'custom_property'). Deprecated in Py...
```

I think better would be better to do:
```
Optional[List[Union[SurfaceFields, str]]]
```

Reply:
Sure. Just FYI this does not have hints either:
![Screenshot 2024-07-09 094808](https://github.com/flexcompute/Flow360/assets/106089368/aa3b630d-d486-46ec-8928-de343ecff36c)

Reply:
oh, I see. Interesting why if doesn't show hints...